src/dev.sh

```diff
@@ -304,13 +304,21 @@ function package ()
         tar_name="${agent_pkg_name}.tar.gz"
         echo "Creating $tar_name in ${LAYOUT_DIR}"
         tar -czf "${tar_name}" -C ${LAYOUT_DIR} .
+        echo "Calculate SHA256 checksum for $tar_name"
+        if [[ ("$PLATFORM" == "linux") ]]; then
+            sha256sum "${tar_name}" | cut -b 1-64 > ./Checksum_SHA256.txt
+        else
+            openssl dgst -sha256 "${tar_name}" | tail -c 64 > ./Checksum_SHA256.txt
+        fi    
     elif [[ ("$PLATFORM" == "windows") ]]; then
         zip_name="${agent_pkg_name}.zip"
         echo "Convert ${LAYOUT_DIR} to Windows style path"
         window_path=${LAYOUT_DIR:1}
         window_path=${window_path:0:1}:${window_path:1}
         echo "Creating $zip_name in ${window_path}"
         powershell -NoLogo -Sta -NoProfile -NonInteractive -ExecutionPolicy Unrestricted -Command "Add-Type -Assembly \"System.IO.Compression.FileSystem\"; [System.IO.Compression.ZipFile]::CreateFromDirectory(\"${window_path}\", \"${zip_name}\")"
+        echo "Calculate SHA256 checksum for $zip_name"
+        powershell -NoLogo -Sta -NoProfile -NonInteractive -ExecutionPolicy Unrestricted -Command "\$(Get-FileHash -Path .\\$zip_name -Algorithm SHA256).Hash | Out-File .\\Checksum_SHA256.txt"
```

Comment:
Is Get-FileHash in powershell 5 only? Is there a way to invoke this from a method that is available in .net 4.5 (prereq to build the agent). Get-FileHash is a function (not a cmdlet), so you can see the source via `(get-command get-filehash).definition`.

Should you check the exit code of powershell and fail?

It would be better to add this into the prior powershell.exe invocation so you dont have to load up powershell.exe twice. Separate multiple statements with `;`. It would be better to set $ErrorActionPreference to 'Stop' at the beginning of the command. powershell.exe will exit with a non-zero exit code if an exception bubbles. Fyi the leading $ (in front of the open parentheses) isn't required but it doesn't hurt anything either. It's mainly useful for when you're inside of a string.